### DISCUSSION TITLE
Admin login

### DISCUSSION THREAD

**Discussion by QPRocky** [strapi/strapi#8683] [Legacy Topics]
Hi, is there way to customize admin login? I see in docs, that you can customize admin/src but problem is that controller I would like to modify, lives in strapi-admin/controllers/authentication.js. 

Basically I would like to following:
1. POST request /admin/login arrives to server
2. make POST request to www.my-external-api.com/login
3. if success, create new strapi admin (if not already exists) and proceed strapi's normal login process

I managed to do this with strapi beta. Just needed to add Auth controller to api folder, but this isnt working anymore.

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
